File: src/models/kanbanAssignment.ts
export type KanbanAssignmentItem = {
  memo_id: number;
  slug_title: string;
  title: string;
  description?: string | null;
  kanban_status_id?: number | null;
  position?: number | null;
  modified_at: string;
  kanban_id: number;
};

export type KanbanAssignmentEntry = {
  kanban_id: number;
  kanban_name: string;
  kanban_status_id?: number | null;
  kanban_status_name?: string | null;
  kanban_status_color?: string | null;
  position?: number | null;
};

File: src-tauri/src/models/kanban_assignment.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize)]
pub struct KanbanAssignmentItem {
    pub memo_id: i32,
    pub slug_title: String,
    pub title: String,
    pub description: Option<String>,
    pub kanban_status_id: Option<i32>,
    pub position: Option<i64>,
    pub modified_at: String,
    pub kanban_id: i32,
}

#[derive(Serialize, Deserialize)]
pub struct KanbanAssignmentEntry {
    pub kanban_id: i32,
    pub kanban_name: String,
    pub kanban_status_id: Option<i32>,
    pub kanban_status_name: Option<String>,
    pub kanban_status_color: Option<String>,
    pub position: Option<i64>,
}
